Update uninstalling-che.adoc (#2887)
codenvy/che-docs: modules/administration-guide/pages/uninstalling-che.adoc

The `--delete-all` option removes the {devworkspace} Operator and the related resources.
====

[IMPORTANT]
====

> Standard operating procedure (SOP) for removing {devworkspace} Operator manually without `{orch-cli}` can be found in the OpenShift Container Platform link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/web_console/web-terminal#removing-devworkspace-operator_uninstalling-web-terminal[official documentation].

Reviewer: ```suggestion
Standard operating procedure (SOP) for removing {devworkspace} Operator manually without `{orch-cli}` is available in the OpenShift Container Platform link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/web_console/web-terminal#removing-devworkspace-operator_uninstalling-web-terminal[official documentation].
```
